app/js/lib/foxbox/db.js

```diff
@@ -65,7 +65,6 @@ export default class Db {
     if (fromVersion < 1) {
       let store = db.createObjectStore(DB_SERVICE_STORE, { keyPath: 'id' });
       store.createIndex('id', 'id', { unique: true });
-      store.createIndex('type', 'type', { unique: false });
```

Comment:
I didn't write an upgrade script. This means users will have to call `foxbox.clear()` when this patch lands.
Let me know if you think upgrade should be considered.